docusaurus/docs/reference/ha/migrating.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_label: Migrating
+sidebar_position: 30
+---
+
+# Migrating Controllers
+
+A controller can be moved from standalone mode to HA mode. It can also be returned
+from HA mode back to standalone mode.
+
+## Standalone to HA
+
+### Requirements
+First, ensure that the controller's certificates and configuration meet the requirements
+in [Bootstrapping](./bootstrapping.md).
+
+### Data Model Migration
+The controller's data can be imported in one of two ways:
```

Comment:
Is there a benefit from supporting two ways? It just seems like a maintenance issue to allow two different ways. Can we remove one or the other? Are there reasons why I would do one or the other? I'd rather give people one way to do it and document that one way. This makes me think there are reasons to support both -- and if there are still compelling reasons, the differences/positives/drawbacks need to be doc'ed

Reply:
It's very minimal code to support both, since they mostly do the same thing. 

I'll kick this over to ops again, because I don't know if it's worth supporting both.

Reply:
Ops agreed we should remove it, so it's removed :+1: